### ISSUE TITLE
Quick Format: Cmd/Ctrl-↓ can't be used immediately after selecting item

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#4191]
After the accessibility/focus changes, you now have to press left-arrow to refocus the bubble before you can press Cmd/Ctrl-↓ to open the citation-options panel. This is a bit of a usability regression, and it means that the guidance panel description ("After you've made your selection, click the bubble or press Ctrl-↓…") isn't even really accurate anymore.

Not sure what to do about this. It makes sense why it doesn't work, but maybe we could maybe cheat and accept Cmd/Ctrl-↓ immediately after an item selection the same way we're accepting a page number even though the bubble isn't technically selected? We'd just close the reference panel and show the options dialog instead.

**Comment by abaevbog** [comment]
During the initial big accessibility-focused rewrite, it was sometimes hard to tell which behaviors were desired features and which were just dictated by the `contenteditable` setup that did not allow things to get properly focused. I must have misidentified `Ctrl-ArrowDown` behavior as the later and it was indeed removed.

Right now, when a bubble is focused, just `arrowDown` opens the options dialog (`ctrl/cmd` is not required). That was done mainly to avoid conflicts with screen readers (https://github.com/zotero/zotero/pull/3464#issuecomment-1804816874 mentions this among other things), as JAWS uses `Ctrl` as a modified for many of its own shortcuts. And, besides, when a bubble is focused, the reference panel gets hidden, so it felt like there's no more need to use a modifier key.

Since this is a behavior we want to keep, I think we could just add `ctrl/cmd-arrowDown` as a general keypress handler to all inputs that will open up the options dialog of the bubble to the left of it. It would not work in cases when most screen readers are active but I think it's not an issue here, since focusing a bubble and pressing `arrowDown` is what the `aria-description` components instruct one to do anyways.

**Comment by dstillman** [comment]
Oh, I see. That might actually be fine — instead of a complicated, undiscoverable (if you don't read the guidance panel) modifier-key shortcut, you can just press left-arrow + down-arrow. That's arguably not really much different.

**Comment by abaevbog** [comment]
Right, discoverability was certainly another consideration as while working through the QF I also struggled with finding out all the available options. So the only reason to bring back `ctrl-arrowDown` now in this form, as far as I can see, is just as a legacy functionality that folks may be used to. I did not see but were there any reports of people trying to use the old shortcut and being disappointed that it doesn't work anymore?

Otherwise, we could just update the guidance panel?

**Comment by dstillman** [comment]
> Otherwise, we could just update the guidance panel?

Yeah, that's fine.
